emocean/Resources/CheckinObservation/CheckinFeelingsView.swift

```diff
@@ -0,0 +1,242 @@
+//
+//  CheckinFeelingsView.swift
+//  emocean
+//
+//  Created by Wilson Adrilia on 03/08/2021.
+//
+
+import SwiftUI
+
+struct CheckinFeelingsView: View {
+    @State var question: String = "How do you feel?"
+    @State var selection: String = "Select Time"
+    @State var energy: CGFloat = 0.0
+    @State var pleasentness: CGFloat = 0.0
+    
+    @State var name: String = "Annoyed"
+    @State var description: String = "Feeling or showing angry irritation or anything that makes you angry"
+    @State var image: String = ""
+    
+    var array: [String] = ["This Morning", "This Afternoon", "This Evening", "This Night"]
```

Comment:
rename variable ini ke variable yang lebih memiliki makna
```suggestion
    var timeDivision: [String] = ["This Morning", "This Afternoon", "This Evening", "This Night"]
```